packit_service/worker/helpers/testing_farm.py

```diff
@@ -614,6 +614,14 @@ def is_fmf_configured(self) -> bool:
         except FileNotFoundError:
             return False
 
+    @staticmethod
+    def is_compose_matching(compose_to_check, composes) -> bool:
+        """
+        Check whether the compose matches any compose in the list of re-compiled
+        composes.
+        """
+        return any(compose.fullmatch(compose_to_check) for compose in composes)
```

Comment:
Could you please add the type annotations? I think it might be handy to get the info about types right there.